broker/utils.go

```diff
@@ -429,6 +429,15 @@ func compactData(entity map[string]interface{}, context interface{}) (interface{
 	return compacted, err
 }
 
+func clearTimeZone(element map[string]interface{}) map[string]interface{} {
```

Comment:
method name should be  removeSystemAppendedTime

Reply:
Done

Reply:
Done